cfgov/v1/atomic_elements/organisms.py

```diff
@@ -1061,9 +1062,10 @@ def links(self):
         for hyperlink in self.get('links') or []:
             url = hyperlink.get('url')
             text = hyperlink.get('text')
+            aria_label = hyperlink.get('aria_label')
 
             if url and text:
-                links.append({'url': url, 'text': text})
+                links.append({'url': url, 'text': text, 'aria_label': aria_label})
```

Comment:
This should satisfy the linter as well:

```suggestion
                links.append({
                    'url': url, 
                    'text': text, 
                    'aria_label': aria_label
                })
```